docs/docs/adding-tags-and-categories-to-blog-posts.md

```diff
@@ -142,8 +144,8 @@ export const pageQuery = graphql`
       edges {
         node {
           frontmatter {
-            title
             path
+            tags
```

Comment:
this seems wrong - we use `title` in this example and don't use `tags`, I think this part shouldn't be changed

Reply:
I just review the docs, and replicated issue #9607 . I retract my PR. Thanks @pieh .